napari/layers/labels/labels.py

```diff
@@ -785,17 +796,41 @@ def color_mode(self):
 
         DIRECT allows color of each label to be set directly by a color dict.
         """
+        warnings.warn(
+            trans._(
+                'Labels.color_mode is deprecated since 0.4.19 and will be '
+                'removed in 0.5.0. Please check type(Labels.colormap) '
+                'instead. napari.utils.colormaps.LabelColormap corresponds to '
```

Comment:
```suggestion
                'instead. napari.utils.colormaps.CycleLabelColormap corresponds to '
```
Right? LabelColormap is the thing being deprecated?